### PR TITLE
docs(readme): fix collectionFormat example (#636)

### PR DESCRIPTION
**Pull request by imjuni**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -488,6 +488,17 @@ You can also apply different serialization `style` and `explode` as specified [h
...
 
 ```js
+// Need to add a collectionFormat keyword to ajv in fastify instance
+const fastify = Fastify({
+  ajv: {
+    plugins: [
+      function (ajv) {
+        ajv.addKeyword({ keyword: 'collectionFormat' })
+      }
+    ]
```
Comment (Eomm) [inline README.md:498]: This should work too

```suggestion
    customOptions: {
      keywords: [ 'collectionFormat' ]
    }
```

Could you add a demo in the https://github.com/fastify/fastify-swagger/tree/master/examples folder?

**Comment by imjuni** [comment]
@Eomm Hi, Thank you for your feedback.

I added addKeyword option on collection-format.js examples.

**Comment by mcollina** [comment]
Linting is failing, could you fix it?

**Comment by imjuni** [comment]
@mcollina Thank you, I fix lint error

File: README.md (hunk 1 above)
Comment (imjuni) [reply README.md]: @Eomm I change plugins > customOptions and demo add examples folder :)

**Review by mcollina — APPROVED** [review]
lgtm

**Review by RafaelGSS — APPROVED** [review]

**Pull request merged**